master/buildbot/test/fake/httpclientservice.py

```diff
@@ -57,6 +57,7 @@ class HTTPClientService(service.SharedService):
 
     getName from the fake and getName from the real must return the same values.
     """
+    quiet = False
```

Comment:
I think this and related changes should be a separate commit: otherwise it's a bit difficult to see what was introduced for the new functionality and what was otherwise required *but* unrelated